Misc changes

rrtoledo opened the pull request:
## Content

This PR includes:
- Moving /docs to .gitignore
- Moving Hashes in utils
- Fixing clippy

## Pre-submit checklist

- Branch
    - [x] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
- PR
    - [x] No clippy warnings in the CI
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested
- Documentation
    - [x] Update README file (if relevant)
    - [x] Update documentation website (if relevant)

## Comments

Misc changes from  from former #66 and #116

## Issue(s)

Relates to #59

curiecrypt reviewed (dismissed):
LGTM 👍 
One thing: I have changed the `DATA_LENGTH = 48`.
However, @tolikzinovyev said: If the data length is changed, it should be done in example PRs, [see](https://github.com/cardano-scaling/alba/pull/120#pullrequestreview-2551082789).
If we go with the way that Tolik suggested, I will re-set the data length to 32. If not, you may want to update this branch :)

tolikzinovyev commented on .gitignore:
```diff
@@ -4,3 +4,4 @@ dist-newstyle
 .DS_Store
 target
 Cargo.lock
+docs
```
Why this? `docs` is committed in the repo.

rrtoledo replied:
I thought that docs should be built locally and not committed

tolikzinovyev commented on tests/centralized_telescope.rs:
```diff
@@ -15,12 +15,12 @@ const DATA_LENGTH: usize = 32;
         completeness_param: 10.0,
-        set_size: 80 * set_size / 100,
-        lower_bound: 20 * set_size / 100,
+        set_size: set_size.saturating_mul(80).div_ceil(100),
```
Does this trigger for you when you run clippy? I doesn't on my laptop or in CI, even though it should...

rrtoledo replied:
It did trigger on the CI in the old PR

rrtoledo commented:
> LGTM 👍 One thing: I have changed the `DATA_LENGTH = 48`. However, @tolikzinovyev said: If the data length is changed, it should be done in example PRs, [see](https://github.com/cardano-scaling/alba/pull/120#pullrequestreview-2551082789). If we go with the way that Tolik suggested, I will re-set the data length to 32. If not, you may want to update this branch :)

OK! I have just moved the DIGEST_SIZE (not DATA_LENGTH), I do not want to change either of the values (it was not changed yet on main when I created this PR).

djetchev reviewed (approved):
LGTM!

rrtoledo commented:
Removing doc from gitifnogre before merging
